### PR TITLE
Cypress 10 support

### PR DESCRIPTION
**Pull request by asad9711**

### CODE REVIEW

File: commands/utils/set_args.js [hunk 1]
```diff
@@ -221,6 +221,13 @@ function sync_args_from_cmd(args) {
       lt_config["run_settings"]["ignore_files"] =
         args["ignore_files"].split(",");
     }
+
+    // if reporter_config_file parameter, add it in lt config alongwith a warning on console
+    if (!lt_config["run_settings"]["reporter_config_file"]) {
+      lt_config["run_settings"]["reporter_config_file"] = constants.LT_BASE_REPORTER_CONFIG_FILE_NAME;
```
Comment (japneetlambdatest) [inline commands/utils/set_args.js:227]: We can add the warning message here

File: commands/utils/validate.js [hunk 2]
```diff
@@ -227,14 +238,20 @@ module.exports = validate_config = function (lt_config, validation_configs) {
...
             );
+          }else if (reporter_config.reporterEnabled && reporter_config.reporterEnabled != ""){
+            if (!reporter_config.reporterEnabled.includes("mochawesome")){
+              reject("Error!! Please add mochawesome in reporterEnabled and add the related config");
+            }
           }
         } catch {
           console.log(
-            "reporter_config_file is not parsed, please provide a valid json in Reporter Config"
+            "reporter_config_file can not parsed, please provide a valid json in Reporter Config"
```
Comment (japneetlambdatest) [inline commands/utils/validate.js:248]: "can not be parsed"

File: commands/utils/validate.js [hunk 3]
```diff
@@ -227,14 +238,20 @@ module.exports = validate_config = function (lt_config, validation_configs) {
             reject(
               "Error!! Reporter JSON File has no keys, either remove Key reporter_config_file from lambdatest config or pass some options"
             );
+          }else if (reporter_config.reporterEnabled && reporter_config.reporterEnabled != ""){
+            if (!reporter_config.reporterEnabled.includes("mochawesome")){
+              reject("Error!! Please add mochawesome in reporterEnabled and add the related config");
```
Comment (japneetlambdatest) [inline commands/utils/validate.js:243]: Can we give a warning message instead of directly rejecting the test

Comment (asad9711) [reply commands/utils/validate.js:243]: makes sense

**Review by japneetlambdatest — APPROVED** [review]

**Pull request merged**